db/data/development-data/articles.js

```diff
@@ -333,7 +333,7 @@ module.exports = [
       'https://images.pexels.com/photos/1267320/pexels-photo-1267320.jpeg?w=700&h=700',
   },
   {
-    title: 'The Notorious MSG`s Unlikely Formula For Success',
+    title: 'The Notorious MSG’s Unlikely Formula For Success',
     topic: 'cooking',
     author: 'grumpy19',
     body: "The 'umami' craze has turned a much-maligned and misunderstood food additive into an object of obsession for the world’s most innovative chefs. But secret ingredient monosodium glutamate’s biggest secret may be that there was never anything wrong with it at all.",
```

Comment:
please avoid typing 'git add .' when adding files to commits - you'll end up sending files that you don't need. Please list out the files instead of the '.' :)